cirq/linalg/decompositions.py

```diff
@@ -102,59 +102,34 @@ def _group_similar(items: List[T],
     return groups
 
 
-def _perp_eigendecompose(matrix: np.ndarray,
-                         rtol: float = 1e-5,
-                         atol: float = 1e-8,
-                         ) -> Tuple[np.array, List[np.ndarray]]:
-    """An eigendecomposition that ensures eigenvectors are perpendicular.
-
-    numpy.linalg.eig doesn't guarantee that eigenvectors from the same
-    eigenspace will be perpendicular. This method uses Gram-Schmidt to recover
-    a perpendicular set. It further checks that all eigenvectors are
-    perpendicular and raises an ArithmeticError otherwise.
+def unitary_eig(matrix: np.ndarray,
+                check_preconditions: bool = True,
+                atol: float = 1e-8) -> Tuple[np.array, np.ndarray]:
+    """Gives the guaranteed unitary eigendecomposition of a unitary matrix.
```

Comment:
"...of a normal matrix."

You can also expand on this in the body of the docstring by adding the definition (it's short enough) and a remark that both unitary and Hermitian matrices are normal. If you like you can also give a counterexample, e.g. creation and annihilation operators are not normal.

(Applies also to Args section below.)